fedora/src/startup/vnc_startup.sh

```diff
@@ -93,41 +93,41 @@ chmod 600 "${PASSWD_PATH}"
 
 XSTARTUP_FILE="${HOME}/.vnc/xstartup"
 if [[ ! -f "${XSTARTUP_FILE}" ]] ; then
-    echo ; echo "Preparing VNC server configuration files ..."
+    echo ; echo "[VNC] Preparing VNC server configuration files ..."
     vncserver "${DISPLAY}"
     vncserver -kill "${DISPLAY}"
-    echo "Saving default startup script as ${XSTARTUP_FILE}.old"
     cp "${XSTARTUP_FILE}" "${XSTARTUP_FILE}.old"
     echo "Replacing default startup script ${XSTARTUP_FILE}"
     cat <<'EOF' > "${XSTARTUP_FILE}"
 #!/bin/sh
 unset SESSION_MANAGER
 unset DBUS_SESSION_BUS_ADDRESS
+# unlock keyring
+echo -n ${VNC_PW} | gnome-keyring-daemon --unlock
 startxfce4 &
```

Comment:
unlock keyring before `startxfce4`.
Also, is it needed to start secrets service with `gnome-keyring-daemon --start --components=secrets` or desktop-client will handle it?

CC @fmoc